CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,4 @@
+Code of Conduct
+===============
+
+Libero follows the [Libero Community Code of Conduct](https://libero.pub/code-of-conduct). 
```

Comment:
Feels quite wordy. 'The Libero community follows a Code of Conduct'?

Reply:
Yes, unless intentional `Code of Conduct` should be understandable